### PR TITLE
Include accepted MIME types in amp-cache-guidelines.md

### PR DESCRIPTION
**Pull request by ericlindley-g**
cc/ @pietergreyling @vitaliybl @bpaduch  @dvoytenko @cramforce @rudygalfi

### CODE REVIEW

File: spec/amp-cache-guidelines.md [hunk 1]
```diff
@@ -56,6 +56,27 @@ Provide guidelines to external parties for implementing their own AMP Caches.
...
+
+Accepted MIME types for images include all `image/` subtypes (e.g. `image/gif`).
+
+### MIME types for fonts
+
+Accepted MIME types for fonts include all `font/` subtypes (e.g. `font/opentype`), as well as the following types.
+
+| Media type / subtype  |
+| ------------- |
+|  application/font   |
```
Comment (cramforce) [inline spec/amp-cache-guidelines.md]: I believe these are all prefixes. You can just put `font/` in the table and say they are prefixes.

Comment (ericlindley-g) [inline spec/amp-cache-guidelines.md]: Good call—updated. Can potentially simplify by not using the wildcard notation and example, but seemed helpful.

File: spec/amp-cache-guidelines.md [hunk 2]
```diff
@@ -56,6 +56,28 @@ Provide guidelines to external parties for implementing their own AMP Caches.
...
 
+## Guidelines: Accepted MIME types
+
+### MIME types for images
+
+Accepted MIME types for images include all `image/` subtypes (e.g. `image/gif`).
+
+### MIME types for fonts
+
+Accepted MIME type prefixes for fonts include the following:
```
Comment (cramforce) [inline spec/amp-cache-guidelines.md]: …the following prefixes.

File: spec/amp-cache-guidelines.md [hunk 3]
```diff
@@ -56,6 +56,28 @@ Provide guidelines to external parties for implementing their own AMP Caches.
...
+
+| Media type / subtype  |
+| ------------- |
+|  font/*   (e.g. `font/opentype`)|
```
Comment (cramforce) [inline spec/amp-cache-guidelines.md]: Remove `*`

The others are technically also prefixes.

E.g. `application/font-foo` is supportred.

File: spec/amp-cache-guidelines.md (hunk 2 above)
Comment (ericlindley-g) [inline spec/amp-cache-guidelines.md]: Added "prefixes" earlier, but I can see how it's clearer at the end of the sentence. Changing now

File: spec/amp-cache-guidelines.md (hunk 3 above)
Comment (ericlindley-g) [inline spec/amp-cache-guidelines.md]: Got it—thanks (that example helps clarify)

**Pull request merged**